e2e/specs/attachments.spec.ts

```diff
@@ -36,7 +36,7 @@ test('Add and remove an attachment', async ({ page }) => {
     await page.click('.cds--file-browse-btn');
   });
 
-  await test.step('And I add the attachment description', async () => {
+  await test.step('And I add a description for the image I uploaded', async () => {
```

Comment:
@denniskigen it's too late that I noticed this change otherwise the wording should be adjusted to `And I add a description for the image to upload'.

Reply:
@denniskigen I've fired a follow-up commit at https://github.com/openmrs/openmrs-esm-patient-chart/pull/1758